pkg/distributor/distributor_test.go

```diff
@@ -3602,6 +3593,10 @@ type prepConfig struct {
 
 	timeOut            bool
 	circuitBreakerOpen bool
+
+	// Ingest storage specific configuration.
+	ingestStorageEnabled    bool  // Whether ingest storage is enabled.
```

Comment:
```suggestion
	ingestStorageEnabled    bool
```

this doesn't say much more than the field name

Reply:
Done in https://github.com/grafana/mimir/pull/7376/commits/a0b5c87d5fc60e1a528eb61e5765bde6e16837e8